### PR TITLE
Make linux_install.sh work with openSUSE Leap 15.0

### PR DESCRIPTION
**Pull request by shieldsd**

### CODE REVIEW

File: util/linux_install.sh [hunk 1]
```diff
@@ -96,9 +96,13 @@ elif grep ID /etc/os-release | grep -q gentoo; then
 	fi
 
 elif grep ID /etc/os-release | grep -qE "opensuse|tumbleweed"; then
+	CROSS_AVR_GCC=cross-avr-gcc8
```
Comment (drashna) [inline util/linux_install.sh]: would `cross-avr-gcc` work better here?

Comment (drashna) [reply util/linux_install.sh]: Sorry, no, I mean for the actual package name.

Comment (shieldsd) [reply util/linux_install.sh]: I don't know about tumbleweed, but openSUSE 15 has no cross-avr-gcc package available in the default repositories.

Comment (shieldsd) [reply util/linux_install.sh]: cross-avr-gcc8 was picked by whoever added the openSUSE support

Comment (drashna) [reply util/linux_install.sh]: Okay, thanks for the clarification.  That's basically what I was asking.

**Pull request merged**